clients/client-python/gravitino/filesystem/gvfs.py

```diff
@@ -756,12 +768,42 @@ def _strip_storage_protocol(storage_type: StorageType, path: str):
         :param storage_type: The storage type
         :param path: The path
         :return: The stripped path
+
+        We will handle OSS differently from S3 and GCS, because OSS has different behavior than S3 and GCS.
```

Comment:
You'd better explain the difference behavior clearly, otherwise someone may not be able to spot the difference at first glance when looking at the examples.

Reply:
Add sentence 
```text
Please take a look at the above example: if we do not remove the protocol (starts with oss://),
it will always return an empty array when we call `oss.ls`, however, if we remove the protocol,
it will return the correct result as expected.
```